js-exercises/exercises.js

```diff
@@ -0,0 +1,182 @@
+/*
+  Reverse a string
+  1. Split the string into an array
+  2. Reverse the array
+  3. Join the array back into a string
+*/
+
+const reverseString = (str) => {
+  return str.split("").reverse().join("");
+};
+
+console.log(`1) Reverse a String: hello = ${reverseString("hello")}`);
+
+/* 
+  Find the largest number in an array
+  1. Sort the array
+  2. Return the last element in the array
+*/
+
+const findLargestNumber = (arr) => {
+  return arr.sort()[arr.length - 1];
+};
+
+console.log(
+  `2) Find largest Number: [6, 3, 2, 7, 5, 3] = ${findLargestNumber([
+    6, 3, 2, 7, 5, 3,
+  ])}`
+);
+
+/*
+  Is a string palindrome?
+  1. Reverse the string
+    a. Split the string into an array
+    b. Reverse the array
+    c. Join the array back into a string
+  2. Compare the reversed string to the original string
+*/
+
+const isPalindrome = (str) => {
+  return str === str.split("").reverse().join("");
+};
+
+console.log(`3) Is Palindrome: racecar = ${isPalindrome("racecar")}`);
+console.log(`3) Is Palindrome: abca = ${isPalindrome("abca")}`);
+
+/*
+  Function to check if two arrays are equal
+  1. Check if the arrays are the same length
+  2. Check if the arrays have the same elements in the same order
+*/
+
+const areArraysEqual = (arr1, arr2) => {
+  if (arr1.length !== arr2.length) return false;
```

Comment:
First you can check if arr1 === arr2 as a short circuit true evaluation

Reply:
Added the above